halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/aws/AwsAddAccountCommand.java

```diff
@@ -34,48 +35,70 @@ protected String getProviderName() {
   }
 
   @Parameter(
-      names = "--default-key-pair",
-      description = AwsCommandProperties.DEFAULT_KEY_PAIR_DESCRIPTION
+    names = "--default-key-pair",
+    description = AwsCommandProperties.DEFAULT_KEY_PAIR_DESCRIPTION
   )
   private String defaultKeyPair;
 
   @Parameter(
-      names = "--edda",
-      description = AwsCommandProperties.EDDA_DESCRIPTION
+    names = "--edda",
+    description = AwsCommandProperties.EDDA_DESCRIPTION
   )
   private String edda;
 
   @Parameter(
-      names = "--discovery",
-      description = AwsCommandProperties.DISCOVERY_DESCRIPTION
+    names = "--discovery",
+    description = AwsCommandProperties.DISCOVERY_DESCRIPTION
   )
   private String discovery;
 
   @Parameter(
-      names = "--account-id",
-      description = AwsCommandProperties.ACCOUNT_ID_DESCRIPTION
+    names = "--account-id",
+    description = AwsCommandProperties.ACCOUNT_ID_DESCRIPTION,
+    required = true
```

Comment:
I guess the only thing I can think of is that we in some hypothetical scenario could use AWS account lookup to find the account id, but I don't think we should support that.